app/models/peep.rb

```diff
@@ -0,0 +1,9 @@
+class Peep
+
+  include DataMapper::Resource
+
+    property  :id,   Serial
+    property  :text, String
+    property  :time, Time
```

Comment:
Unnecessary spacing detected.<br>Inconsistent indentation detected.<br>Put one space between the method name and the first argument.